docs/github.md

```diff
@@ -11,25 +11,26 @@ authenticated http.
 
 ## Basic workflow
 
-The simplest way to start with Jujutsu, is creating a stack of commits, before
-creating any branch.
+The simplest way to start with Jujutsu is to create a stack of commits first.
+You will only need to create a branch when you need to push the stack to a 
+remote. 
 
 ```shell script
-# Start a new commit off of `main`
+# Start a new commit off of the default branch.
 $ jj new main
 # Refactor some files, then add a description and start a new commit
 $ jj commit -m 'refactor(foo): restructure foo()'
 # Add a feature, then add a description and start a new commit
 $ jj commit -m 'feat(bar): add support for bar'
 # Create a branch so we can push it to GitHub
-$ jj branch create bar -r @-
+$ jj branch create bar -r @- # create a branch `bar` that now contains the previous two commits.
 # Push the branch to GitHub (pushes only `bar`)
 $ jj git push
 ```
 
 While it's possible to create a branch and commit on top of it in a Git like
```

Comment:
Extra-optional: I'd put a dash in "Git-like".

Reply:
done